playground/backend/containers/go/Dockerfile

```diff
@@ -55,6 +55,7 @@ ENV SERVER_PORT=8080
 ENV APP_WORK_DIR=/opt/playground/backend/
 ENV BEAM_SDK="SDK_GO"
 ENV BUCKET_NAME="playground-precompiled-objects"
+ENV PROPERTY_PATH=/opt/playground/backend/properties.yaml
```

Comment:
PROPERTY_PATH was added for every docker file to avoid problems when we will use properties.yaml for runners (JAVA, PYTHON, SCIO, GO)